packages/react-native-codegen-typescript-test/package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "@react-native/codegen-typescript-test",
   "version": "0.0.1",
-  "description": "⚛️ TypeScript related unit test for @react-native/codegen",
```

Comment:
Using symbols like ⚛️ in description is just extra noise for npm and other registries, that too at the start of the descriptions string.
Removing it should make the search easier.

Same applies for all files.